tsconfig.json

```diff
@@ -15,6 +15,8 @@
     "**/*.tsx",
     "**/*.js",
     "**/*.jsx",
+    "emotion.d.ts",
```

Comment:
These should already be included with `"**/*.ts"` 🤔

Reply:
I know. For those specific types, I just prefer to be "extra" explicit.